<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidebarWidget</class>
 <widget class="QWidget" name="SidebarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ÎèÑÍµ¨ Ìå®ÎÑê</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: #3c3f41; color: white;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>ÎèÑÍµ¨ ÏÑ§Ï†ï</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="toggleButton">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ìå®ÎÑê Ï†ëÍ∏∞/ÌéºÏπòÍ∏∞</string>
        </property>
        <property name="text">
         <string>‚óÄ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="iconBarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #3c3f41;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="cameraButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Ïπ¥Î©îÎùº ÏÑ§Ï†ï</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4e5254;
    color: white;
    border: none;
    border-radius: 3px;
    padding: 5px;
}
QPushButton:checked {
    background-color: #6a6a6a;
}
QPushButton:hover {
    background-color: #5c5c5c;
}</string>
           </property>
           <property name="text">
            <string>üì∑</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modelButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Î™®Îç∏ ÏÑ§Ï†ï</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4e5254;
    color: white;
    border: none;
    border-radius: 3px;
    padding: 5px;
}
QPushButton:checked {
    background-color: #6a6a6a;
}
QPushButton:hover {
    background-color: #5c5c5c;
}</string>
           </property>
           <property name="text">
            <string>üîç</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="toolsButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>ÎèÑÍµ¨</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4e5254;
    color: white;
    border: none;
    border-radius: 3px;
    padding: 5px;
}
QPushButton:checked {
    background-color: #6a6a6a;
}
QPushButton:hover {
    background-color: #5c5c5c;
}</string>
           </property>
           <property name="text">
            <string>üîß</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="cameraPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="cameraSettingsGroup">
            <property name="title">
             <string>Ïπ¥Î©îÎùº ÏÑ§Ï†ï</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="cameraLabel">
               <property name="text">
                <string>Ïπ¥Î©îÎùº:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cameraCombo"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="refreshButton">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>‚Üª</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="resolutionLabel">
               <property name="text">
                <string>Ìï¥ÏÉÅÎèÑ:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QComboBox" name="resolutionCombo"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="fpsLabel">
               <property name="text">
                <string>FPS:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QSpinBox" name="fpsSpinBox">
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="saveSettingsCheckBox">
            <property name="text">
             <string>Í∏∞Î≥∏ ÏÑ§Ï†ïÏúºÎ°ú Ï†ÄÏû•</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="connectButton">
              <property name="text">
               <string>Ïó∞Í≤∞</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="disconnectButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Ïó∞Í≤∞ Ìï¥Ï†ú</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="modelPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="detectionGroup">
            <property name="title">
             <string>Í∞ùÏ≤¥ ÌÉêÏßÄ ÏÑ§Ï†ï</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="detectionCheckBox">
               <property name="text">
                <string>Í∞ùÏ≤¥ ÌÉêÏßÄ ÌôúÏÑ±Ìôî</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="modeLabel">
                 <property name="text">
                  <string>ÌÉêÏßÄ Î™®Îìú:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="modeCombo">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>OpenCV-HOG</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>YOLO</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="yoloGroup">
            <property name="title">
             <string>YOLO ÏÑ§Ï†ï</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="versionLabel">
                 <property name="text">
                  <string>YOLO Î≤ÑÏ†Ñ:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="versionCombo"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="modelLabel">
                 <property name="text">
                  <string>Î™®Îç∏:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="modelCombo"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="confidenceLabel">
                 <property name="text">
                  <string>Ïã†Î¢∞ÎèÑ:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="confidenceSlider">
                 <property name="maximum">
                  <number>99</number>
                 </property>
                 <property name="value">
                  <number>40</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="confidenceValueLabel">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0.40</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="applyButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>ÏÑ§Ï†ï Ï†ÅÏö©</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="currentSettingsGroup">
            <property name="title">
             <string>ÌòÑÏû¨ ÏÑ§Ï†ï</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="currentSettingsLabel">
               <property name="text">
                <string>ÌÉêÏßÄÍ∏∞Í∞Ä ÏÑ§Ï†ïÎêòÏßÄ ÏïäÏïòÏäµÎãàÎã§.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolsPage">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="toolsGroup">
            <property name="title">
             <string>Í∏∞ÌÉÄ ÎèÑÍµ¨</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="objectSelectionButton">
               <property name="text">
                <string>ÌÉêÏßÄ Í∞ùÏ≤¥ ÏÑ†ÌÉù</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="recordingButton">
               <property name="text">
                <string>ÎÖπÌôî ÏÑ§Ï†ï</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>